source/2019-12-02-countdown-to-the-new-year-ember-modifier.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I think to reflect the correct date on the blog, this file would need to be named `source/2019-12-03-countdown-to-the-new-year-ember-modifier.md` - what do you think?

Reply:
oh good catch! silly mistake!